submodules/LIB/hardware/modules/iob_interface/README.md

```diff
@@ -1,3 +1,30 @@
-# IOb-interface #
+# IOb-interface
 
-![Read Transaction](iob_if_read.png "Read Transaction")
+The IOb-interface has the following signals:
```

Comment:
The IOb Native Interface, or IOb interface shortly, has the following signals from the slave perspective. (Note that the _i and _o suffixes denote input and output, respectively.)